Habitica/src/main/java/com/habitrpg/android/habitica/models/inventory/Mount.java

```diff
@@ -44,6 +45,18 @@ public void setType(String type) {
         this.type = type;
     }
 
+    @Override
+    public String getTranslatedType(Context c) {
+        String currType = type;
+        String translation = c.getString(R.string.drop);
+
+        if (currType.equals(translation)) {
+            currType = c.getString(R.string.standard);
+        }
+
+        return currType;

```

Comment:
The way this currently works is that it will get the _translated_ name of `drop`. With that the comparison would not work in all cases, since if the user is using the app in p.e. german, the value of `translation` would not be `drop` anymore. Instead the comparison should just happen with the string `"drop"`, so that it works in all languages.

Can you also add a mapping for the following:
"premium" -> "Magic Potion"
"quest" -> "Quest"
"wacky" -> "Wacky"
"special" -> "Special"

With the values it maps to all being translated the way you currently have with the `standard`. The reason for also mapping the quest, wacky and special is that with this, they can also be translated into other languages.